.travis.yml

```diff
@@ -17,7 +17,7 @@ matrix:
         - python: 2.7
           env: TOXENV=qa
 
-install: pip install -U setuptools pip tox coveralls
+install: pip install -IU setuptools pip tox coveralls
```

Comment:
whats `I`?

Reply:
maybe use long versions instead, e.g. `--ignore-installed` and `--upgrade` @rm-hull